### DISCUSSION TITLE
Logical operators in keys section of custom-dictionaries

### DISCUSSION THREAD

**Discussion by IgorSh16** [akvorado/akvorado#1427] [Q&A]
Hi Vincent,
I'm trying to find out how to mark client-to-server and server-to-client flows in the table
Is there a way to use logical operators in keys section of custom-dictionaries ?
My CSV file  have information about pairs of server IPs and destination ports separated by comma. It would be perfect if I can use logical operator "AND" for two keys - DstAddr and DstPort. If DstAddr and DstPort both match, then to mark flow as client-to-server in new column of flow table.

**Comment by vincentbernat** [comment]
Currently, this is not possible as this is not implemented. We can't put anything too complex there as this needs to be translated into a dictionary lookup in ClickHouse.
